content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Master Controller Cheat-Sheet"
+linkTitle: "Cheat-Sheet MasterController"
+
+weight: 800
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available commands below.
+---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands). by executing all commands automatically.
+
+### Example(Windows PowerShell)
+
+- This would display all the available command options
```

Comment:
see comments [here](https://github.com/Xceptance/xlt-documentation/pull/100) for shell example, same same